Source/Tests/CMakeLists.txt

```diff
@@ -67,13 +67,47 @@ endif()
 install(PROGRAMS "${PROJECT_SOURCE_DIR}/Scripts/FEXUpdateAOTIRCache.sh" DESTINATION bin RENAME FEXUpdateAOTIRCache)
 
 if (CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  # Check for conflicting binfmt before installing
+  set (CONFLICTING_BINFMTS_32
```

Comment:
Also, if this suggestion is adopted, we maybe can just move the script invocation into the `binfmt_misc_64` CMake targets themselves as an extra `COMMAND` argument. Though you'd have to check if CMake properly cancels the command chain in case of an error. (I would expect it does, but then again it's CMake.)

Reply:
There we go, let the script handle multiple arguments for checking.
Still split between x86 and x64 since it is valid to install only 32-bit or 64-bit depending on needs.

Script invocation moved in to the add_custom_target invocation because yes it does properly cancel the chain when there is an error.

Reply:
Cheers, new version looks great 👍